tests/extensions/data/comment_ending.py

```diff
@@ -0,0 +1,7 @@
+"""comment-ending test-case"""
+A = 5  #
+A = A + 1
```

Comment:
```suggestion
#
A = A + 1
######################################
# Example of comment used as delimiter
######################################
```
First line with only a comment should be an error for sure. But what do you think about the line full of `#`, should it be an error ?

Reply:
Line with only single `#` - should raise a message.
Row of comments - shouldn't raise a message. I think many people might use titles like that. My purpose was not to check for **lines with only comments in it**. I want it to detect when the comment is empty. There is much less ambiguity in a 'comments row' like that.  Imagine a collaboration where this pattern is used for separating sections, they will get False Positives everywhere running Pylint -> They will turn this checker off -> Empty comments would not be detected..

Reply:
And as a note - checking if a line as a row of only `#` symbols can be implemented as simple as:
`all( char=='#' for char in line)`

Reply:
Fair enough, let's separate the two issues